test/Tryouts/Corax/IndexReader.cs

```diff
@@ -131,8 +132,8 @@ internal static unsafe string GetExternalId(TransactionOperationContext context,
                 {
                     ThrowBadData();
                 }
-
-                return Encoding.UTF8.GetString(tvr.Read(2, out var size), size);
+                
+                return new LazyStringValue(null,tvr.Read(2, out var size), size, context);
```

Comment:
Use the context for this. We have a cache exactly for this

Reply:
https://github.com/ravendb/ravendb/blob/v4.0/src/Sparrow/Json/JsonOperationContext.cs#L94

Reply:
Oh. Will do.